ports/sentry-native/vcpkg.json

```
@@ -27,6 +27,11 @@
         "Cf. https://github.com/getsentry/sentry-native#compile-time-options"
       ],
       "dependencies": [
+        {
+          "$comment": "pkgcong is used by the breakpad backend.",
+          "name": "pkgconf",
+          "platform": "!android & !ios"
```

```suggestion
          "name": "pkgconf",
          "platform": "!android & !ios",
          "host": true
```
But remember to run `vcpkg format-manifest ...`.